src/lib/cfa/ntp_conf.rb

```diff
@@ -346,13 +356,32 @@ def options_matcher
     class CommandRecord < Record
       def options
         return [] unless tree_value?
-        augeas_options.map { |option| option[:key] }
+        augeas_options.each_with_object({}) do |option, opts|
+          opts[option[:key]] = option[:value]
+        end.to_a.flatten.compact
       end
 
       def options=(options)
         ensure_tree_value
         tree_value.tree.delete(options_matcher)
-        options.each { |option| tree_value.tree.add(option, nil) }
+
+        key = nil
+
+        # Some options can be a key value pair or just a single one
+        options.each do |option|
+          if KEY_VALUE_CMD_OPTIONS.include?(option)
+            key = option
+          else
+            add_option(key, option)
+            key = nil
+          end
+        end
```

Comment:
@jreidinger this was the first option I was considering but according to documentation:

https://www.eecis.udel.edu/~mills/ntp/html/confopt.html#option

It should not be a valid entry. 

But if you consider it more readable, then for me is enough to rewrite it ;)